src/test/java/com/team3/central/services/EventServiceTest.java

```diff
@@ -247,4 +263,98 @@ void getForUser() {
     // then
     assertThat(res).isNotEmpty();
   }
+
+  @Test
+  public void deleteEvent_shouldReturnFalse_whenEventNotFound() {
+    // given
+    final String ORGANIZER_EMAIL = "organizer@example.com";
+    final Long EVENT_ID = 1L;
+    when(eventRepository.findById(anyLong())).thenReturn(Optional.empty());
+
+    // when
+    boolean result = eventService.deleteEvent(EVENT_ID, ORGANIZER_EMAIL);
+
+    // then
+    assertFalse(result);
+  }
+
+  @Test
+  public void deleteEvent_shouldReturnFalse_whenOrganizerEmailMismatch() {
+    // given
+    final String ORGANIZER_EMAIL = "organizer@example.com";
+    final Long EVENT_ID = 1L;
+    Event event = new Event();
+    OrganizerEntity organizer = new OrganizerEntity();
+    organizer.setEmail("other@example.com");
+    event.setOrganizer(organizer);
+    when(eventRepository.findById(anyLong())).thenReturn(Optional.of(event));
+
+    // when
+    boolean result = eventService.deleteEvent(EVENT_ID, ORGANIZER_EMAIL);
+
+    // then
+    assertFalse(result);
+  }
+
+  @ParameterizedTest
+  @MethodSource("testData")
+  public void deleteEvent_shouldReturnFalse_whenEventAlreadyCancelled(EventStatus status) {
+    // given
+    final String ORGANIZER_EMAIL = "organizer@example.com";
+    final Long EVENT_ID = 1L;
+    Event event = new Event();
+    event.setStatus(status);
+    OrganizerEntity organizer = new OrganizerEntity();
+    organizer.setEmail(ORGANIZER_EMAIL);
+    event.setOrganizer(organizer);
+    when(eventRepository.findById(anyLong())).thenReturn(Optional.of(event));
+
+    // when
+    boolean result = eventService.deleteEvent(EVENT_ID, ORGANIZER_EMAIL);
+
+    // then
+    assertFalse(result);
+  }
+
+  @Test
+  public void deleteEvent_shouldCancelEvent_whenValidRequest() {
+    // given
```

Comment:
We need to make naming convention consistent, change this naming style here or for all tests please